File: src/client_gen/java/Debug.java
import java.io.IOException;
import java.io.OutputStream;

public class Debug {
    private OutputStream stream;

    public Debug(OutputStream stream) {
        this.stream = stream;
    }

    public void send(model.DebugCommand command) {
        try {
            new model.ClientMessage.DebugMessage(command).writeTo(stream);
            stream.flush();
        } catch (IOException e) {
            throw new RuntimeException(e);
        }
    }
}

File: src/client_gen/kotlin/Debug.kt
import java.io.IOException
import java.io.OutputStream

class Debug(private val stream: OutputStream) {

    fun send(debugData: model.DebugData) {
        try {
            model.ClientMessage.DebugDataMessage(debugData).writeTo(stream)
            stream.flush()
        } catch (e: IOException) {
            throw RuntimeException(e)
        }
    }
}
